.github/label-actions.yml

```diff
@@ -1,21 +1,31 @@
 'reproduction:needed':
   comment: >
-    This issue needs to be reproduced in order to continue.
+    Hi there,
 
-    Please consult our guide on [minimal reproductions](https://github.com/renovatebot/renovate/blob/master/docs/development/minimal-reproductions.md) to understand what is needed.
+    The Renovate team needs your help!
+    To fix the problem, we first need to know exactly what's causing the bug.
+    A minimal reproduction help us to pinpoint the exact cause of the bug.
 
-    Issues requiring reproductions may be closed if no reproduction can be provided within two weeks.
+    To get started, please read our guide on [minimal reproductions](https://github.com/renovatebot/renovate/blob/master/docs/development/minimal-reproductions.md) to understand what is needed.
+
+    We may close the issue if you have not provided a minimal reproduction within two weeks.
+    If you need more time, or are stuck, please ask for help or more time in a comment.
+
+    Good luck,
+
+    The Renovate team
 
 'reproduction:provided':
   comment: >
-    Thank you for providing a reproduction.
+    Thank you for providing a reproduction! :tada: :rocket:
 
-    The label will be updated to `reproduction:confirmed` once someone else has confirmed they can reproduce the problem with it.
+    The Renovate team will take a look at the reproduction repository.
+    Once we confirm the provided repository reproduces the problem, the label will be changed to `reproduction:confirmed`.
 
 'status:requirements':
   comment: >
-    This issue has been labeled with `status:requirements`. This indicates that the issue is not ready to start and further requirements or reproduction are needed first.
+    This issue has been labeled with `status:requirements`. This indicates that we don't know enough to start work and further requirements or a reproduction are needed first.
 
-    This label should be removed and replaced with `status:ready` once all requirements and reproductions necessary to start have been met.
+    This label will be replaced with `status:ready` once all requirements and reproductions necessary to start work have been met.
 
-    If it's *unclear* as to what is missing to move this issue forward, or you think it has already been provided, feel free to nudge a maintainer to take a look and clarify.
+    If it's *unclear* as to _what_ is missing to move this issue forward, or if you think it has already been provided, feel free to nudge a maintainer (how????) to take a look and clarify.
```

Comment:
Should probably explain how to "nudge a maintainer".
Do we want them to use `@rarkins` or `@renovate-bot` or something else?

Reply:
Maybe instead say to ask in a comment

Reply:
I've pushed 2 new commits to split up this sentence and explain that the user should create a comment.